python/ray/data/dataset.py

```diff
@@ -4336,7 +4336,8 @@ def _warn_slow(self):
         if ray.util.log_once("dataset_slow_warned"):
             logger.warning(
                 "The `map`, `flat_map`, and `filter` operations are unvectorized and "
-                "can be very slow. Consider using `.map_batches()` instead."
+                "can be very slow. If you are using a vectorizable transformation, "
+                "consider using `.map_batches()` instead."
```

Comment:
```suggestion
                "can be very slow. If you're using a vectorized transformation, "
                "consider using `.map_batches()` instead."
```

Nit: Use contractions. They're easier to read.


The word is vectorized, right? Don't think vectorizable is a word.

Reply:
updated